frontend/src/components/pages/your_clubs/ClubList.tsx

```diff
@@ -1,25 +1,25 @@
 import * as React from "react";
+import { BookInfo } from "../club_display/BookInfo";
 import { BookInterface } from "../../../services/backend_service_interface/backend_service_interface";
 import Box from "@material-ui/core/Box";
 import Button from "@material-ui/core/Button";
+import { ClubDescription } from "../club_display/ClubDescription";
 import { ClubInterface } from "../../../services/backend_service_interface/backend_service_interface";
+import { ContentWarnings } from "../club_display/ContentWarnings";
 import { createStyles } from "@material-ui/core/styles";
 import Dialog from "@material-ui/core/Dialog";
 import DialogActions from "@material-ui/core/DialogActions";
 import DialogContent from "@material-ui/core/DialogContent";
 import DialogContentText from "@material-ui/core/DialogContentText";
 import DialogTitle from "@material-ui/core/DialogTitle";
 import HighlightOffIcon from "@material-ui/icons/HighlightOff";
+import { Link } from "react-router-dom"
```

Comment:
```suggestion
import { Link } from "react-router-dom";
```
Semicolon at end of import.